.storybook/main.js

```diff
@@ -42,19 +39,4 @@ module.exports = {
       },
     },
   ],
-  webpackFinal: async (config) => {
-    const SVGSpritesPlugin = new SVGSpritemapPlugin(
-      path.resolve(__dirname, '../src/icons/**/*.svg'),
-      {
-        sprite: {
-          prefix: false,
-          generate: {
-            symbol: true,
-          },
-        },
-      },
-    );
-    config.plugins.push(SVGSpritesPlugin);
-    return config;
-  },
```

Comment:
Oooh is the storybook startup faster now?

Reply:
Not that I can tell, but I wasn't real scientific about it.